tests/integration/test_task.py

```diff
@@ -80,3 +83,229 @@ def _run(self, input_files):
     print(order)
     assert len(order) == 12
     assert order == sorted(order)
+
+
+@pytest.mark.parametrize('executables', ['ls', 'mv'])
+def test_which(executables):
```

Comment:
there is no need to test this function anymore, because now it's just imported from the shutil library instead of having our own implementation

Reply:
yarp, good call, removed it - I was looking at it in a dubious manner yesterday but then I forgot I actually put it there myself :beer: